target/linux/ath79/generic/base-files/etc/board.d/01_leds

```diff
@@ -91,6 +91,10 @@ comfast,cf-e314n-v2)
 	ucidef_set_led_rssi "rssimediumhigh" "RSSIMEDIUMHIGH" "$boardname:green:rssimediumhigh" "wlan0" "51" "100"
 	ucidef_set_led_rssi "rssihigh" "RSSIHIGH" "$boardname:green:rssihigh" "wlan0" "76" "100"
 	;;
+comfast,cf-e375ac)
+	ucidef_set_led_netdev "lan" "LAN" "$board:green:lan" "eth0"
+	ucidef_set_led_switch "lan" "LAN" "$board:green:lan" "switch0" "0x04"
```

Comment:
Honestly?

Reply:
I've forgotten to remove the netdev line

Reply:
Depending on what OEM firmware does, consider whether this should be limited to lan-port or show activity for both ports (0x6) ...

Reply:
After reflash, I can confirm that the commit for the ar71xx did the led assignments wrong. 

Red is for the WAN port 
Green for the LAN port 
and blue for phy1pt.

`root@COMFAST:/sys/class/leds# ls -la
drwxr-xr-x    2 root     root             0 Jan  1  1970 .
drwxr-xr-x   19 root     root             0 Jan  1  1970 ..
lrwxrwxrwx    1 root     root             0 Aug 27 22:33 ath9k-phy1 -> ../../devices/platform/qca956x_wmac/leds/ath9k-phy1
lrwxrwxrwx    1 root     root             0 Jan  1  1970 comfast:blue:ext_lan -> ../../devices/platform/leds-gpio/leds/comfast:blue:ext_lan
lrwxrwxrwx    1 root     root             0 Jan  1  1970 comfast:blue:ext_wan -> ../../devices/platform/leds-gpio/leds/comfast:blue:ext_wan
lrwxrwxrwx    1 root     root             0 Jan  1  1970 comfast:blue:ext_wlan -> ../../devices/platform/leds-gpio/leds/comfast:blue:ext_wlan
lrwxrwxrwx    1 root     root             0 Jan  1  1970 comfast:blue:wlan -> ../../devices/platform/leds-gpio/leds/comfast:blue:wlan
lrwxrwxrwx    1 root     root             0 Jan  1  1970 comfast:green:lan -> ../../devices/platform/leds-gpio/leds/comfast:green:lan
lrwxrwxrwx    1 root     root             0 Jan  1  1970 comfast:red:wan -> ../../devices/platform/leds-gpio/leds/comfast:red:wan
root@COMFAST:/sys/class/leds# cat comfast\:blue\:wlan/
brightness      max_brightness  trigger
device/         subsystem/      uevent
root@COMFAST:/sys/class/leds# cat comfast\:blue\:wlan/trigger 
none switch0 netdev phy0rx phy0tx phy0assoc phy0radio phy0tpt phy1rx phy1tx phy1assoc phy1radio [phy1tpt] 
root@COMFAST:/sys/class/leds# cat comfast\:red\:wan/trigger 
none [switch0] netdev phy0rx phy0tx phy0assoc phy0radio phy0tpt phy1rx phy1tx phy1assoc phy1radio phy1tpt 
root@COMFAST:/sys/class/leds# cat comfast\:red\:wan/port_mask 
0x2
root@COMFAST:/sys/class/leds# cat comfast\:green\:lan/trigger 
none [switch0] netdev phy0rx phy0tx phy0assoc phy0radio phy0tpt phy1rx phy1tx phy1assoc phy1radio phy1tpt 
root@COMFAST:/sys/class/leds# cat comfast\:green\:lan/port_mask 
0x4
`